### PR TITLE
Back-end and Databse for SignUp and Login.

### PR DESCRIPTION
**Pull request by SomendraZ**
I made and connected back-end for user signup and login and its working properly.
First I was using axios but it was not working properly so @PiyushNegi1703 help me and we changed axios to fetch api.

### CODE REVIEW

File: Front-End/src/Components/Login.js [hunk 1]
```diff
@@ -1,15 +1,47 @@
...
-    navigate("/Discover");
-  }
+
+  const login = async () => {
+    const email = document.getElementById("emailLogin").value;
+    const password = document.getElementById("passwordLogin").value;
+    const rememberMe = document.getElementById("check").checked;
+  
+    try {
+      const response = await fetch("http://localhost:6969/api/user/login", {
```
Comment (PatelNeelMahesh) [inline Front-End/src/Components/Login.js]: Add `http://localhost:6969` to `.env` and use here using `process.env`.

File: Front-End/src/Components/SignUp.js [hunk 2]
```diff
@@ -1,14 +1,72 @@
...
+    const passwordRegex = /^(?=.*[A-Z])(?=.*[!@#$&*])(?=.{8,})/;
+    if (!passwordRegex.test(password)) {
+      alert(
+        "Password must be at least 8 characters long and contain at least one uppercase letter and one special character"
+      );
+      return;
+    }
+  
+    try {
+      const response = await fetch("http://localhost:6969/api/user/", {
```
Comment (PatelNeelMahesh) [inline Front-End/src/Components/SignUp.js]: Add `http://localhost:6969` to `.env` and use here using`process.env`.

**Review by PatelNeelMahesh — COMMENTED** [review]
After making the required changes you can merge pull request.

File: Front-End/src/Components/SignUp.js (hunk 2 above)
Comment (SomendraZ) [reply Front-End/src/Components/SignUp.js]: done

File: Front-End/src/Components/Login.js (hunk 1 above)
Comment (SomendraZ) [reply Front-End/src/Components/Login.js]: done

**Pull request merged**